google/resource_redis_instance.go

```diff
@@ -673,10 +661,6 @@ func expandRedisInstanceTier(v interface{}, d *schema.ResourceData, config *Conf
 	return v, nil
 }
 
-func expandRedisInstanceRegion(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
-	return v, nil
-}
-
 func resourceRedisInstanceEncoder(d *schema.ResourceData, meta interface{}, obj map[string]interface{}) (map[string]interface{}, error) {
 	delete(obj, "region")
```

Comment:
this custom encoder can probably be deleted now that we don't add it in the first place!